README.md

```diff
@@ -54,6 +54,9 @@ host = "localhost"
 disable = false
 host = "localhost"
 port = 5001
+
+[orderbook]
+internalmatching = true
```

Comment:
We’ll probably gonna need at least another one for `orderbook`, which is the internal matching priority (before or after peerOrders queue). I didn’t want to put it yet, because it felt like it's an overkill at this stage. I can also think on some other stuff too, down the road: memory limit, persistence config, etc.
In addition, I think that removing it from `orderbook` and putting it as a standalone property will make this config name less self explanatory.

But if you think that it will be used in other domains too, then tell me and i’ll take it out.

Reply:
Ah I see. I actually can't think right now of a concrete example where this may affect other domains. But the idea is that, if this option is enabled, `xud` acts as the sole orderbook for the entire exchange, right? It seems like it may have broader impact, but maybe I am mistaken and if that does become the case we can deal with it later.

As for the internal matching priority, we can discuss more later but it seems to me like internal matching would always be prioritized. Internal matches are instant and guaranteed, and the internal exchange can claim both sides of the trading fee, whereas swaps with external exchanges may fail or have minor delays.

For now I figure we can have this configuration option either way. One idea that will at least make it shorter is to call it `orderbook.internal`, which also conveys that the orderbook is being used as the internal orderbook for the exchange. Up to you.

Reply:
It seems to me that `orderbook.internal` will be too confusing, although it captures the greater meaning of this configuration. Not sure if people will easily understand what it means by looking at the code or config.
I think i’ll just leave it as it is now, and we’ll do the necessary changes later on.

In regards to priorities - I actually made internal matching on lower priority. I will fix it.